specification/windowsesu/resource-manager/Microsoft.WindowsESU/preview/2019-09-16-preview/windowsesu.json

```diff
@@ -0,0 +1,571 @@
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "MultipleActivationKey": {
+      "description": "MAK key details.",
+      "type": "object",
+      "required": [
+        "location"
+      ],
+      "properties": {
+        "id": {
+          "description": "Resource Id.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Name of the MAK key.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Resource type.",
+          "type": "string",
+          "readOnly": true
+        },
+        "location": {
+          "description": "Resource location.",
+          "type": "string",
+          "x-ms-mutability": ["create", "read"]
+        },
+        "tags": {
+          "description": "Resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "description": "MAK key specific properties.",
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "properties": {
+            "multipleActivationKey": {
+              "description": "MAK 5x5 key.",
+              "type": "string",
+              "readOnly": true
+            },
+            "expirationDate": {
+              "description": "End of support of security updates activated by the MAK key.",
+              "format": "date-time",
+              "type": "string",
+              "readOnly": true
+            },
+            "osType": {
+              "description": "Type of OS for which the key is requested.",
+              "type": "string",
+              "enum": [
+                "Windows7",
+                "WindowsServer2008",
+                "WindowsServer2008R2"
+              ],
+              "x-ms-enum": {
+                "name": "OsType",
+                "modelAsString": true
+              },
+              "x-ms-mutability": ["create", "read"]
+            },
+            "supportType": {
+              "description": "Type of support",
+              "type": "string",
+              "enum": [
+                "SupplementalServicing",
+                "PremiumAssurance"
+              ],
+              "default": "SupplementalServicing",
+              "x-ms-enum": {
+                "name": "SupportType",
+                "modelAsString": true
+              },
+              "x-ms-mutability": ["create", "read"]
+            },
+            "installedServerNumber": {
+              "description": "Number of activations/servers using the MAK key.",
+              "type": "integer",
```

Comment:
This is user set or read-only?

Reply:
Set by user on creation, immutable, as most of other properties.